modules/users/client/components/AboutMe.component.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import sanitizeHtml from 'sanitize-html';
+import { withNamespaces } from '@/modules/core/client/utils/i18n-angular-load';
+{/* import { Trans } from 'react-i18next'; */}
+import '@/config/lib/i18n';
```

Comment:
This is called in the `i18n-angular-load`, but does no harm here. So it can stay. (Just saying.)